exercises/python/encryption-context-complete/src/document_bucket/__init__.py

```diff
@@ -1,12 +1,30 @@
 import os
-import sys
```

Comment:
wait, why is this the same as #55?

Reply:
Relative to `master` it includes the same commits? I'm hurrying a tad and not being super careful with my PR commits. Make sense or did I misgrok the concern?